.github/workflows/e2e-test.yaml

```diff
@@ -91,6 +91,10 @@ jobs:
           - arch: amd64
```

Comment:
Can we also remove the TODO immediately above this?

Reply:
yep, rephrased to describe the current state rather then intention